ecs-cli/modules/cli/local/down_app.go

```diff
@@ -65,19 +87,17 @@ func downComposeLocalContainers() error {
 	return nil
 }
 
-func downAllLocalContainers() error {
+func downLocalContainersWithFilters(args filters.Args) error {
 	ctx, cancel := context.WithTimeout(context.Background(), docker.TimeoutInS)
 	defer cancel()
 
 	client := docker.NewClient()
 	containers, err := client.ContainerList(ctx, types.ContainerListOptions{
-		Filters: filters.NewArgs(
-			filters.Arg("label", taskDefinitionLabelKey),
-		),
-		All: true,
+		Filters: args,
+		All:     true,
 	})
 	if err != nil {
-		logrus.Fatalf("Failed to list containers with label=%s due to %v", taskDefinitionLabelKey, err)
+		logrus.Fatalf("Failed to list containers with label=%s due to %v", taskDefinitionLabelValue, err)
```

Comment:
Should we list all `args` here instead of just the `taskDefinitionLabelValue` ?
```suggestion
		logrus.Fatalf("Failed to list containers with filters %v due to %v", args, err)
```

Reply:
Yeah I agree with you.